doc/Caliptra.md

```diff
@@ -120,34 +121,33 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 5. <a id="ref-5"></a>TCG DICE Attestation Architecture Version 1.00 Revision 0.23 March 1, 2021
 6. <a id="ref-6"></a>TCG Hardware Requirements for a Device Identifier Composition Engine Family “2.0” Level
 7. <a id="ref-7"></a>[Side-Channel Attacks: Ten Years After Its Publication and the Impacts on Cryptographic Module Security Testing](https://csrc.nist.gov/csrc/media/events/physical-security-testing-workshop/documents/papers/physecpaper19.pdf)
-8. <a id="ref-8"></a>[Attestation V1.0 White Paper](https://www.opencompute.org/documents/attestation-v1-0-20201104-pdf)
-9. <a id="ref-9"></a>[Attestation of System Components v1.0 Requirements and Recommendations](https://www.opencompute.org/documents/attestation-v1-0-20201104-pdf)
-10. <a id="ref-10"></a>OCP Security WG: Ownership Transfer
-11. <a id="ref-11"></a>Ownership and Control of Firmware in Open Compute Project Devices (Open Compute Project, Security WG)
+8. <a id="ref-9"></a>[Attestation of System Components v1.0 Requirements and Recommendations](https://www.opencompute.org/documents/attestation-v1-0-20201104-pdf)
+9. <a id="ref-10"></a>OCP Security WG: Ownership Transfer
+10. <a id="ref-11"></a>Ownership and Control of Firmware in Open Compute Project Devices (Open Compute Project, Security WG)
```

Comment:
Ref 11 can be removed, we can merge 10 and 11. We don't have a public spec yet for this reference either, so perhaps should be removed altogether.

Reply:
Out of scope for this commit. Please file an issue.